command/server/generate_config.go

```diff
@@ -0,0 +1,45 @@
+package server
```

Comment:
This file should be in a separate package within the `server` command directory, like `export` (see commands like `txpool`)

Reply:
Moved to dedicated package 9350d7d3c963f0b39070691337bd8610bbf3da8a